### PR TITLE
fix: address credo complaint

### PR DESCRIPTION
**Pull request by nlwstein**
#### Summary of changes

Potential resolution to credo complaint in https://github.com/mbta/arrow/pull/971

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)
- [ ] This branch was deployed to the staging environment and is currently running with no unexpected increase in warnings, and no errors or crashes.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -42,7 +42,7 @@ defmodule Arrow.MixProject do
   # Specifies your project dependencies.
   #
   # Type `mix help deps` for examples and options.
-  defp deps do
```
Comment (meagharty) [inline mix.exs:45]: I noticed this too!

This private function is called above on line 13 in `def project`, it looked like it was done that way in other mbta repos too. I'm not sure if it's an outdated convention but I don't feel strongly about it (changing it or not).

Comment (meagharty) [reply mix.exs:45]: Should have asked: why did you decide to change it?

Comment (meagharty) [reply mix.exs:45]: Oh, if it removes a compilation warning then sounds okay to :shipit:

**Comment by nlwstein** [comment]
> I had a question re: `defp/def deps` but this looks good to me, thank you!

I'll let you merge or copy / paste whatever you want from this, unless you want to treat it like a normal PR. Let me know if you want me to merge etc.

**Pull request merged**